vancouver-tree-height-geo.Rmd

```diff
@@ -45,11 +45,25 @@ head(trees)
 
 ### Cleaning the Data
 
-We are only interested in neighbourhood names and tree heights, so we should make sure there are no missing values in these columns
+We are primarily interested in the NEIGHBOURHOOD_NAME, HEIGHT_RANGE, and HEIGHT_RANGE_ID columns, so it is crucial to ensure that there are no missing values in these columns to maintain the accuracy and reliability of our analysis. Upon checking, we confirmed that none of these three columns contain any missing data. Additionally, we examined the entire dataset for duplicate records using the unique identifier TREE_ID and confirmed that there are no duplicate rows. This step ensures that no records are inadvertently counted multiple times.
 
 ```{r}
-str(trees) # maybe we don't use this
-# look at NEIGHBOURHOOD_NAME, HEIGHT_RANGE, HEIGHT_RANGE_ID
+# str(trees) # maybe we don't use this
+# Check for missing data in NEIGHBOURHOOD_NAME, HEIGHT_RANGE, HEIGHT_RANGE_ID columns
+missing_data_check <- trees |>
+  summarise(
+    NEIGHBOURHOOD_NAME_missing = sum(is.na(NEIGHBOURHOOD_NAME)),
+    HEIGHT_RANGE_missing = sum(is.na(HEIGHT_RANGE)),
+    HEIGHT_RANGE_ID_missing = sum(is.na(HEIGHT_RANGE_ID))
+  )
+print(missing_data_check) # no missing data in the columns of interest
+
+# Check for duplicates in data set
```

Comment:
Same here, "Checking for duplicates...." also could be displayed as a subtitle.

Reply:
it is the comment line as well, but I made some edits to make it clearer